src/Device.ts

```diff
@@ -52,16 +52,25 @@ class Main {
 
         // Barebone implementation of the On/Off cluster
         const onOffClusterServer = new ClusterServer(OnOffCluster,
-            { on: false }, // Off by default
             {
-                on: async ({attributes: {on}}) => on.set(true),
-                off: async ({attributes: {on}}) => on.set(false),
-                toggle: async ({attributes: {on}}) => on.set(!on.get()),
+                onOff: false, // Off by default
+                globalSceneControl: true,
```

Comment:
The current typing requires to also set values for all optional fields ... So each implementation needs at least to have something for these values, also if they would never use them. WOuld it make sense to also allow these attributes here to be "type wise" optional?

Reply:
![Bildschirmfoto 2022-10-21 um 22 00 22](https://user-images.githubusercontent.com/11976694/197279293-8823245a-2fba-44ba-be91-a04b593faa89.png)

This is what my WebStorm marks
![Bildschirmfoto 2022-10-21 um 22 01 28](https://user-images.githubusercontent.com/11976694/197279419-a933a533-d6b7-4a0b-823e-8970940d1c5a.png)

Teh the tsc build tells this

Reply:
So I can not compile the TS without the values

Reply:
I could add a TODO and you check it after merge because in fact it is ok to have the values there was just wondering that I need to